adafruit_rfm9x.py

```diff
@@ -523,10 +534,11 @@ def rssi(self):
         # Remember in LoRa mode the payload register changes function to RSSI!
         return self._read_u8(_RH_RF95_REG_1A_PKT_RSSI_VALUE) - 137
 
-    def send(self, data):
+    def send(self, data, timeout_s=2.):
```

Comment:
No `_s`. Just document that its in seconds. This should be our standard for timeouts. (`time.sleep` is an example of the same units.)

Reply:
OK - this was just a holdover from the original code. The change makes sense to me.